pkg/apis/policy/v1alpha1/propagation_types.go

```diff
@@ -44,6 +44,8 @@ const (
 // +genclient
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 // +kubebuilder:resource:path=propagationpolicies,scope=Namespaced,shortName=pp,categories={karmada-io}
+// +kubebuilder:printcolumn:JSONPath=`.spec.conflictResolution`,name="Conflict-Resolution",type=string
+// +kubebuilder:printcolumn:JSONPath=`.spec.priority`,name="Priority",type=string
```

Comment:
Is it possible to set `priority=1` for these two values, so that they are visible when `-o wide` is specified?

Reply:
Echo from [kubebuilder document](https://book.kubebuilder.io/reference/markers/crd), the priority indicates
> Lower priority (higher numbered) columns will be hidden if the terminal width is too small.

So, they will be visible as long as the terminal width is sufficient. 

We can set priority after we have more columns.

Reply:
that's a good suggestion, will apply

Reply:
Just run a test with this:
```
[rainbowmango]# kubectl get propagationpolicies.policy.karmada.io nginx-propagation 
NAME
nginx-propagation
[rainbowmango]# 
[rainbowmango]# kubectl get propagationpolicies.policy.karmada.io nginx-propagation -o wide
NAME                CONFLICT-RESOLUTION   PRIORITY
nginx-propagation   Abort                 0
```
The new columns will not be shown even the terminal width is large enough. They can be shown with `-o wide`.

@XiShanYongYe-Chang @a7i Is this behavior you want?

I think the columens is helpful, it shouldn't be hidden by default, given they are all we got by now.

Reply:
It seems that the default AGE column is missing.

My understanding is that the two new columns are not necessarily necessary for users viewing Policy. If enough users want to view the column directly (without adding -o wide), we can adjust its priority.

Reply:
> It seems that the default AGE column is missing.

That's bad, let me fix this.

Reply:
Sent the PR #5138.